LayoutTests/imported/w3c/web-platform-tests/background-fetch/content-security-policy.https.window.js

```diff
@@ -13,8 +13,11 @@ meta.setAttribute('http-equiv', 'Content-Security-Policy');
 meta.setAttribute('content', "connect-src 'none'");
 document.head.appendChild(meta);
 
-backgroundFetchTest((t, bgFetch) => {
+backgroundFetchTest(async (t, bgFetch) => {
+  const fetch = await bgFetch.fetch(uniqueId(), '/');
+
+  const record = await fetch.match('/');
```

Comment:
Please export this change

Reply:
Yes, this is https://github.com/web-platform-tests/wpt/pull/38741.
The export tool should probably tag the corresponding PR automatically in addition to the bugzilla entry.